src/app/ui/overrides/adf-document-list.theme.scss

```diff
@@ -123,5 +99,25 @@
     padding-top: 9px;
     padding-bottom: 9px;
     min-height: auto;
+
+    .adf-datatable-cell-value {
```

Comment:
can this be moved to ADF?

Reply:
yes, the plan is to move all these styles to ADF with [ACA-2361](https://issues.alfresco.com/jira/browse/ACA-2361)